### PR TITLE
[Feature]: Add dropdown menu in title view containing link to settings and changelog

### LINKED ISSUE #255: Enable a changelog button on the target list
Enable a link on the "target list" panel that redirects you to the changelog post in the blog.

### PR DESCRIPTION
**Pull request by mliao95**
This PR adds a dropdown menu in the extension's title view that contains links to the extension's settings and changelog.

GIF:
extension-more-actions

closes #255

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -437,6 +446,11 @@
                     "command": "vscode-edge-devtools-view.refresh",
                     "when": "view == vscode-edge-devtools-view.targets",
                     "group": "navigation"
+                },
+                {
```
Comment (joselea) [inline package.json:450]: what is the difference between these 2 commands? the registration that has a title and icon and the second one that has a when and a group?

**Review by joselea — COMMENTED** [review]
LGTM, have a quick question

File: package.json (hunk 1 above)
Comment (mliao95) [reply package.json:450]: The first section with the title and icon is the actual registration of the command whereas the second one is to place the existing command to the extension title bar like this:
image

**Review by joselea — APPROVED** [review]
LGTM

**Review by vidorteg — CHANGES_REQUESTED** [review]
I like the change but I'm not sure if the toolbar is the best place for it, as it feels out of place with the other buttons (which are extension commands). I took a look to other extensions and some of them add a "..." menu at the end which triggers a context menu where we could add the changelog (along with maybe other options like "Settings"). similar to this:
image

**Review by vidorteg — APPROVED** [review]
Looks incredible @mliao95 !, thanks for the quick turnaround

**Pull request merged**